release/create_candidate_pr.py

```diff
@@ -0,0 +1,79 @@
+# pylint: disable=redefined-outer-name, invalid-name
+
+"""
+Creates a new release candidate branch and makes a pull request
+for it into the 'release' branch.
+"""
+
+from __future__ import print_function
+
+import argparse
+
+from release.get_token import get_token
+from release.github_api import GithubApi, RequestFailed
+from release import utils
+
+
+def _build_parser():
+    """ Creates an option parser for this command """
+    expected_date = utils.default_expected_release_date()
+
+    result = argparse.ArgumentParser(
+        description="""
+        Create a new release candidate branch and associated pull request.
+        """
+    )
+
+    result.add_argument(
+        '--release-date', nargs=1, default=expected_date,
+        help="""
+        Specify a date that the release branch is expected to be deployed.
+        Should be in YYYY-MM-DD format. If not passed, defaults to the
+        next upcoming Tuesday, which is currently {date}.
+        """.format(date=expected_date.date().isoformat()),
+        )
+    result.add_argument(
+        '--org', nargs=1, default='edx',
+        help="""
+        Specify a github organization to work under. Default is 'edx'.
+        """)
+    result.add_argument(
+        '--repo', nargs=1, default='edx-platform',
+        help="""
+        Specify a github repository to work with. Default is 'edx-platform'.
+        """)
```

Comment:
nit: move trailing paren to the next line: https://github.com/edx/edx-platform/wiki/Python-Guidelines#breaking-long-lines